docs/en/index.md

```diff
@@ -0,0 +1,47 @@
+title: Login Forms
+summary: Customising the login forms for a SilverStripe site
+
+# Introduction
+
+The [Login Forms module](https://addons.silverstripe.org/add-ons/silverstripe/login-forms) provides an improved default
+view for the authentication UIs in SilverStripe. Rather than applying the forms to the site's default Page template,
+Login Forms renders a simple, responsive, standalone interface, which can be tweaked to suit an individual SilverStripe
+site but doesn't require any extra work to incorporate the UI into the site's design.
```

Comment:
Do you want to add some info that indicates the goal is to make this login screen match the CMS rather than the website?